plans/feat-1858-keymap-discoverability.md

```
@@ -0,0 +1,539 @@
+
+CodeRabbit が frontmatter の書き込み契約が曖昧だと指摘（「4 つ書く」と「3 つは説明するだけ」が
+同じ文にある）。直そうとして frontmatter をパースしてみたら、**そもそも YAML として読めなく
+なっていた。**
+
+```text
+# mulmoterminal-keys/SKILL.md だけを見た結果。他の skill については下の表のとおり。
+origin/main    => PARSES OK
+HEAD (pushed)  => ERROR: bad indentation of a mapping entry
+```
+
+原因は `: `（コロン+空白）。**引用符の無い YAML スカラーはこれを含められない** —— mapping の
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)